src/bin/adcm_sdk_pack

```diff
@@ -31,14 +31,24 @@ if __name__ == "__main__":
                         help='set splitter release flag to true', default=None)
     parser.add_argument('-e', '--edition', default=None, required=False,
                         help='edition to build, usefull for spec with multiple editions')
+    parser.add_argument('-s', '--no-timestamp', action="store_true",
```

Comment:
Provide a default value

Reply:
And add required=False

Reply:
Added for clarity, but really 'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.